lib/kafka/producer.rb

```diff
@@ -225,22 +268,6 @@ def deliver_messages
       end
     end
 
-    # Returns the number of messages currently held in the buffer.
```

Comment:
IS this code snippet removal related in anyway to this PR? (just to understand if I am missing something in the picture :^))